docs/integrations/aa/candide.md

```diff
@@ -0,0 +1,325 @@
+# Candide
+
+# Recover a Safe Account with Google using AbstractionKit
+
+This is the second of two guides on how to recovery a Safe Account using Google. In the first guide, we show the steps on how to add the recovery method. We recommend starting with the first guide [here](https://docs.candide.dev/wallet/guides/recovery-with-google-using-lit/).
+
+## What is AbstractionKit?
+AbstractionKit is a Typescript Library to easily build on Account Abstraction, with first class support for Safe Accounts. One of their use cases is to enable users to add recovery methods as a backup in case their use their main signer.
```

Comment:
> AbstractionKit is a Typescript Library to easily build on Account Abstraction, with first class support for Safe Accounts. One of their use cases is to enable users to add recovery methods as a backup in case their use their main signer.

Consider: "AbstractionKit is a Typescript Library that enables developers to easily build on Account Abstraction, with first class support for Safe Accounts. One of the unique use cases enabled by AbstractionKit is the ability for users to add a recovery method(s) as a backup to their account in the case that they lose access to their main signing key."